frontend/wallet/src/main/AppWindow.re

```diff
@@ -75,6 +75,11 @@ include Single.Make({
         },
       );
 
+      Bindings.Fs.watchFile(
```

Comment:
nit: Can you stick a comment here about how this is to enable hot reloading

Reply:
Done.